cmd/clusterctl/clusterdeployer/clusterdeployer.go

```diff
@@ -367,6 +367,9 @@ func pivotNamespace(from, to clusterclient.Client, namespace string) error {
 		}
 		glog.Infof("Moved Cluster '%s'", cluster.GetName())
 	}
+	if err := from.DeleteClusterObjectsInNamespace(namespace); err != nil {
```

Comment:
The PR description says "after pivot succeed" but this will clean some parts up in the middle of the pivot.